translate/cloud-client/translate_v3_batch_translate_text_with_glossary_test.py

```diff
@@ -34,10 +34,7 @@ def glossary():
 
     yield glossary_id
 
-    try:
-        translate_v3_delete_glossary.delete_glossary(PROJECT_ID, glossary_id)
-    except Exception:
-        pass
+    translate_v3_delete_glossary.delete_glossary(PROJECT_ID, glossary_id)
```

Comment:
I think what @kurtisvg didn't like was that the `Exception` here is too wide, and lack of reporting the details.

However, I also don't like the new code because the test will fail when cleanup fails.

I think `delete_glossary()` possibly throws the following exceptions:

- `google.api_core.GoogleAPICallError`
- `google.api_core.exceptions.DeadlineExceeded`
- `google.api_core.exceptions.RetryError`
- `google.cloud.exceptions.NotFound`
- `ValueError`

According to [this page](https://googleapis.dev/python/translation/latest/gapic/v3/api.html) and from my experience, since the sample code involves long running operations.

I think you can
- ignore `NotFound`
- raise (or not catch) `RetryError` and `ValueError`
- catch `GoogleAPICallError` and `DeadlineExceeded` then retry with `backoff`

Can you do something like the following?

```python
import backoff
from google.api_core import GoogleAPICallError
from google.api_core.exceptions import DeadlineExceeded
from google.api_core.exceptions import RetryError
from google.cloud.exceptions import NotFound

    # ...
    # cleanup
    @backoff.on_exception(
        backoff.expo, (GoogleAPICallError, DeadlineExceeded), max_time=60)
    def delete_glossary():
        try:
            translate_v3_delete_glossary.delete_glossary(PROJECT_ID, glossary_id)
        except NotFound as e:
            # Ignoring this case.
            print("Got NotFound, detail: {}".format(str(e)))

    delete_glossary()
```

Reply:
Okay will do!